packages/web-react/src/components/Accordion/README.md

```diff
@@ -0,0 +1,111 @@
+# Accordion
+
+## Usage
+
+### Default (Always open)
+
+```javascript
+import React, { useState } from 'react';
+import {
+  Accordion,
+  AccordionItem,
+  AccordionHeader,
+  AccordionContent,
+  AccordionOpenStateType,
+} from '@lmc-eu/spirit-web-react/components';
+```
+
+```
+const [openState, setOpenState] = useState<AccordionOpenStateType | undefined>(undefined);
+```
+
+```javascript
+const toggle = (id: string) => {
+  let value;
+  if (typeof openState === 'string') {
+    if (openState === id) {
+      value = '';
+    } else {
+      value = id;
+    }
+  } else if (Array.isArray(openState)) {
```

Comment:
Why not to use `Array.includes` and `Array.filter`? e. g.

```js
const toggle = (id) => {
    if (Array.isArray(openState)) {
      if (open.includes(id)) {
        setOpen(open.filter((accordionId) => accordionId !== id));
      } else {
        setOpen([...open, id]);
      }
    } else if (open === id) {
      setOpen(undefined);
    } else {
      setOpen(id);
    }
  };
```

Reply:
fixed